cms/static/sass/_build.scss

```diff
@@ -81,3 +81,4 @@
 
 @import 'developer'; // used for any developer-created scss that needs further polish/refactoring
 @import 'shame';     // used for any bad-form/orphaned scss
+@import 'extras'; // extra styles for edraak
```

Comment:
I like your approach, just one final tweak. Remove **extras**, then go to directory: `common/static/sass/edraak`, there you will find a file called `_arabic-mathjax.scss`, next to it and in the same directory, add a file called `_header.scss`, then come back here, and under the line where `arabic-mathjax` is imported, import the header file. This should work and I will approve and merge.